LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -100,7 +100,49 @@ function importTable($db, $tempFolder, $table) {
 importTable($db, $tempFolder, 'workflow_routes');
 importTable($db, $tempFolder, 'step_modules');
 
+if (is_dir($paste_custom_templates) && is_dir($copy_custom_templates)) {
+    // remove files from templates/email/custom_override on subordinate
+    $files = glob($paste_custom_templates . '*');
+
+    // Deleting all the files in the list
+    foreach ($files as $file) {
+        if(is_file($file)) {
+            // Delete the given file
+            unlink($file);
+        }
+    }
+
+    // copy templates/email/custom_override from primary to subordinate
+    copyDirectory($copy_custom_templates, $paste_custom_templates);
+}
+
 $db->query("ALTER TABLE `records_dependencies` ADD CONSTRAINT `fk_records_dependencyID` FOREIGN KEY (`dependencyID`) REFERENCES `dependencies`(`dependencyID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `dependency_privs` ADD CONSTRAINT `fk_privs_dependencyID` FOREIGN KEY (`dependencyID`) REFERENCES `dependencies`(`dependencyID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `category_count` ADD CONSTRAINT `category_count_ibfk_1` FOREIGN KEY (`categoryID`) REFERENCES `categories`(`categoryID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
 $db->query("ALTER TABLE `category_privs` ADD CONSTRAINT `category_privs_ibfk_2` FOREIGN KEY (`categoryID`) REFERENCES `categories`(`categoryID`) ON DELETE RESTRICT ON UPDATE RESTRICT;");
+
+function copyDirectory($source, $destination) {
+    $files = scandir($source);
+
+    foreach ($files as $file) {
+        if ($file !== '.' && $file !== '..') {
+            // not sure if the / is needed here
+            $sourceFile = $source . '/' . $file;
+            $destinationFile = $destination . '/' . $file;
+
+            if (is_dir($sourceFile)) {
+                copyDirectory($sourceFile, $destinationFile);
+            } else {
+                if (is_dir($destinationFile)) {
+                    copy($sourceFile, $destinationFile);
+                } else {
+                    // theoretically this should never be hit. All portals should have the same dir structure
+                    // thereby making this mute, but jsut in case.
+                    mkdir($destinationFile);
+                    copy($sourceFile, $destinationFile);
+                }
+
+            }

```

Comment:
I think your original implementation was fine.

scandir() already ensures you're either getting a directory or a file, and it's not necessary to check again.

```suggestion
            if (is_dir($sourceFile)) {
                copyDirectory($sourceFile, $destinationFile);
            } else {
                copy($sourceFile, $destinationFile);
            }
```
(double check that merging the suggestion worked, this has caused issues before)

Reply:
This has been changed back.